src/Catty/Defines/LanguageTranslationDefines.h

```diff
@@ -223,6 +223,7 @@
 #define kLocalizedMeasure NSLocalizedString(@"Measure", nil)
 #define kLocalizedSize NSLocalizedString(@"Size", nil)
 #define kLocalizedSelectCategories NSLocalizedString(@"Select categories", nil)
+#define kLocalizedSelectCategoriesDescription NSLocalizedString(@"Please select up to three different Categories", nil)
```

Comment:
The label should be named "Please choose up to three categories" (instead of select and small letter for "c") to reuse the translations of Catroid.